components/popover/index.js

```diff
@@ -96,23 +96,39 @@ class Popover extends Component {
 	}
 
 	focus() {
-		const { focusOnMount = true } = this.props;
-		if ( ! focusOnMount ) {
-			return;
+		const { focusOnMount } = this.props;
+
+		// Boolean values for focusOnMount deprecated in 3.2.
+		if ( focusOnMount === true ) {
+			window.console.warn( '<Popover> component: focusOnMount should be false or a string as of Gutenberg 3.2. See: https://github.com/WordPress/gutenberg/tree/master/components/popover/README.md' );
```

Comment:
I think it's worth using `deprecated()` here and adding a note to `docs/deprecated.md`. Then, in 3.4, we can remove the check entirely.

```js
deprecated( 'focusOnMount={ true }', {
	version: '3.4',
	alternative: 'focusOnMount="firstElement"',
	plugin: 'Gutenberg',
} );
```

Reply:
Oh, nice! I am total noob so I didn't know how to deprecate stuff. That's totally slick.